adapter_pipelines/submit.wdl

```diff
@@ -103,6 +103,7 @@ task create_submission {
       --submit_url ${submit_url} \
       --analysis_process_path analysis_process.json \
       --analysis_protocol_path analysis_protocol.json \
+      --outputs_file_path outputs.json \
```

Comment:
What do you think about also adding outputs.json as an output of this task? https://github.com/HumanCellAtlas/pipeline-tools/blob/5b587eda87ff949664eae3031d87e05d4e3f898a/adapter_pipelines/submit.wdl#L117

Reply:
Good catch, I meant to do that but ended up forgetting about it! Will do.